webhook_server/libs/handlers/labels_handler.py

```
@@ -186,7 +274,15 @@ def _get_custom_pr_size_thresholds(self) -> list[tuple[int | float, str, str]]:
 
         if not sorted_thresholds:
             self.logger.warning(f"{self.log_prefix} No valid custom thresholds found, using static defaults")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)